scripts/OWNERS

```diff
@@ -2,11 +2,7 @@
 
 approvers:
 - mohammedzee1000
-- amitkrout
-- cdrage
 
 
 reviewers:
 - mohammedzee1000
```

Comment:
similar to for `tests` directory
Can we include @prietyc123 as approver and @rnapoles-rh as a reviewer?

Reply:
done